src/client/Microsoft.Identity.Client/MsalErrorMessage.cs

```diff
@@ -397,6 +397,8 @@ public static string CertMustBeRsa(string certificateFriendlyName)
 
         public const string ClientCredentialWrongAuthority = "The current authority is targeting the /common or /organizations endpoint which is not recommended. See https://aka.ms/msal-net-client-credentials for more details.";
 
+        public const string OnBehalfOfWrongAuthority = "The current authority is targeting the /common or /organizations endpoint which is not recommended. See https://aka.ms/msal-net-on-behalf-of for more details.";

```

Comment:
```suggestion
        public const string OnBehalfOfWrongAuthority = "The current authority is targeting the /common or /organizations endpoint. Instead, it should target the same tenant as the client, which can be found in the tid claim of client token. See https://aka.ms/msal-net-on-behalf-of for more details.";

```

Reply:
Minor updates to Bogdan's suggestion.
```suggestion
        public const string OnBehalfOfWrongAuthority = "The current authority is targeting the /common or /organizations endpoint. Instead, it should target the same tenant as the client, which can be found in the 'tid' claim of the incoming client token. See https://aka.ms/msal-net-on-behalf-of for more details.";
```